docs/access-control.rst

```diff
@@ -233,15 +233,15 @@ Do a POST with a correct username and password, and you will get a response like
     }
 
 
-POST with a incorrect username and password, and you will get a response like this, with a HTTP status of 400.
+Do a POST with a incorrect username and password, and you will get a response like this, with a HTTP status of 400.
```

Comment:
Revert and update:

POST with an incorrect username and password, and you will get a response like this, with a HTTP status of 400.